app/views/paginable/plans/_index.html.erb

```diff
@@ -36,9 +36,7 @@
           <% if has_dmp_id %>
             <td>
               <% if plan.dmp_id.present? %>
-                <a href="<%= plan.dmp_id.value %>" target="_blank" class="has-new-window-popup-info">
-                  <%= id_for_display(id: plan.dmp_id, with_scheme_name: false).html_safe %>
-                </a>
+                <%= dmp_id_for_display(dmp_id: plan.dmp_id, without_prefix: true).html_safe %>
```

Comment:
I see `id_for_display` still being used in the app. I just wanted to check whether we need to update those cases as well.

Reply:
good question. Those are still appropriate as their identifiers are stored in the join table still, so `id.value` still works for org and user scenarios